### DISCUSSION TITLE
Windows 11 autostart not working

### DISCUSSION THREAD

**Discussion by Varinn35** [Rem0o/FanControl.Releases#578] [Q&A] [answered]
I've been using FanControl without problems on Win10 for some time, but I was just forced to reformat and decided to try Win11 to see how it is. For some reason I cannot get FanControl to start with the OS. Checking the box in the app does nothing as it unticks itself on every restart, adding a shortcut to the startup folder didn't help either. I've setup my configs and the profiles are loading with the program but no matter how I do it, the auto-start feature turns itself off.

On OS boot I need to manually open the app or the fans all run at 100% constantly. I feel like it might be related to the UAC or whatever it's called now, as every time I open the program it prompts for admin access. I can't find any way to permanently allow this one program through this shield to test it.

Am I missing something simple?

Looking in my userconfig file I don't see any reference to the start with windows option

![image](https://user-images.githubusercontent.com/103897629/163726301-569d5091-7d10-4577-83de-39034a2a6e70.png)

**Comment by xhul-dev** [comment, marked as answer]
Hello there.
Try this: https://github.com/xhul-dev/FanControl-WinStart
Note that it was tested under windows 10, so let me know if something doesn't work.
Cheers.

**Reply by Varinn35** [reply]
That solved it!

I set the timer to 15s and it worked like a charm. I'll run it for a few days to be sure but it looks good so far

**Comment by Blackwolf2022** [comment]
the problem also occurs when fancontrol updates, i always solve this way, open fancontrol, uncheck and select these 2 items
Start minimized
Start with Windows
then on the right click on Save configuration!
